### PR TITLE
resource/alicloud_ess_eci_scaling_configuration: update active_deadline_seconds and container_group_name constriction

### PR DESCRIPTION
**Pull request by fuliu-zln**
…ne_seconds and container_group_name constriction

### CODE REVIEW

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Review by ChenHanZhang — COMMENTED** [review]
approved

File: alicloud/resource_alicloud_ess_eci_scaling_configuration.go [hunk 1]
```diff
@@ -57,9 +57,9 @@ func resourceAlicloudEssEciScalingConfiguration() *schema.Resource {
 				Optional: true,
 			},
 			"container_group_name": {
-				Type:     schema.TypeString,
-				Optional: true,
-			},
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validation.StringMatch(regexp.MustCompile(`^[a-z0-9][-a-z-0-9]{2,128}$`), "It must be 2 to 128 characters in length and can contain lowercase letters, digits, and hyphens (-). It must start with a letter or a digit.")},
```
Comment (ChenHanZhang) [inline alicloud/resource_alicloud_ess_eci_scaling_configuration.go:62]: 不用用 validation 包，去掉validation，直接使用 StringMatch

**Comment by ChenHanZhang** [comment]
pr 标题改下，不要驼峰和下划线混用

File: alicloud/resource_alicloud_ess_eci_scaling_configuration.go (hunk 1 above)
Comment (fuliu-zln) [reply alicloud/resource_alicloud_ess_eci_scaling_configuration.go]: done

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Review by ChenHanZhang — COMMENTED** [review]
approved

File: alicloud/resource_alicloud_ess_eci_scaling_configuration_test.go [hunk 2]
```diff
@@ -498,12 +498,12 @@ func TestAccAliCloudEssEciScalingConfigurationBasic(t *testing.T) {
...
-						"active":                           "true",
-						"force_delete":                     "true",
-						"description":                      "newDesc",
-						"security_group_id":                "sg-bp1hi5tpb5c3e51a15pf2",
-						"container_group_name":             "newName",
+						"scaling_configuration_name": "newName",
+						"active":                     "true",
+						"force_delete":               "true",
+						"description":                "newDesc",
+						//"security_group_id":                CHECKSET,
```
Comment (MrWolong) [inline alicloud/resource_alicloud_ess_eci_scaling_configuration_test.go:505]: 注释去掉

Comment (fuliu-zln) [reply alicloud/resource_alicloud_ess_eci_scaling_configuration_test.go:505]: done

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Pull request merged**